src/main/java/com/ctc/wstx/api/WstxInputProperties.java

```diff
@@ -300,6 +300,13 @@ public final class WstxInputProperties
      * DTD subset).
      */
     public final static String P_BASE_URL = "com.ctc.wstx.baseURL";
+    
+    /**
+     * Property of type {@link java.lang.Boolean}, that will allow parsing
+     * high unicode characters written by surrogate pairs (2 code points)
+     * Default set as Boolean.FALSE, because it is not a standard behavior
+     */
```

Comment:
Let's add

    @since 6.6

(next version due to additional functionality and not just bug fixes)